make/photon/db/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM vmware/mariadb-photon:10.2.10
+FROM vmware/mariadb-photon:__version__
```

Comment:
Can we make sure the ` vmware/mariadb-photon:__version__` should never exist?

Reply:
no better solution so far, just leave it to replace the dockerfile.